palisade-service/mvn_dependency_tree.txt

```diff
@@ -248,7 +246,7 @@ org.antlr/stringtemplate/version = 3.2.1
 
 org.apache.avro/avro/version = 1.8.2
 
-org.apache.commons/commons-compress/version = 1.18
+org.apache.commons/commons-compress/version = 1.8.1
```

Comment:
commons-compress version 1.18 looks like the correct one as 1.8.1 was released in May 2014. Can you have a look please why this version has gone backwards.

Reply:
Amended Kubernetes client and mock versions so that they are consistent across all poms.